Cargo.toml

```
@@ -21,7 +21,7 @@ libc = "0.2"
 geo-types = { version = "0.7" }
 gdal-sys = { path = "gdal-sys", version = "^0.8" }
 ndarray = { version = "0.15", optional = true }
-chrono = { version = "0.4" }
+chrono = { version = ">= 0.4.0, <=0.4.22" }
```

Nit, feel free to ignore if you'll change it back in #336:

```suggestion
chrono = { version = ">= 0.4.0, <= 0.4.22" }
```